packages/apps-config/src/api/spec/equilibrium.ts

```diff
@@ -1,164 +1,108 @@
 // Copyright 2017-2023 @polkadot/apps-config authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import type { Observable } from 'rxjs';
-import type { ApiInterfaceRx, AugmentedQuery, RxResult } from '@polkadot/api/types';
-import type { AccountData, AccountId, AccountIndex, Address, Balance } from '@polkadot/types/interfaces';
-import type { Codec, OverrideBundleDefinition } from '@polkadot/types/types';
+import type { ApiInterfaceRx } from '@polkadot/api/types';
+import type { Enum } from '@polkadot/types';
+import type { AccountId, AccountIndex, Address, Balance } from '@polkadot/types/interfaces';
+import type { OverrideBundleDefinition } from '@polkadot/types/types';
+import type { Struct, u64, u128, Vec } from '@polkadot/types-codec';
+import type { ITuple } from '@polkadot/types-codec/types';
```

Comment:
Just add `@polkadot/types-codec` to `package.json`. (Run `yarn` and then add `yarn.lock` as well)